### ISSUE TITLE
Reading in Archives?

### ISSUE THREAD

**Issue by uli-on** [alialbaali/Noto#197]
I can't open archived notes, only select. I believe I remember I _**could**_ before v2.3.0-beta1. Am I wrong?

**Comment by alialbaali** [comment]
Nope, you're not. If you'd like to modify the note, or read it, you have to unarchive first. 

This is part of marking archives simpler, as discussed here #182.

**Comment by uli-on** [comment]
I'd like to outline my deliberations and will come to something a tad off topic. 

------

- In order to read my archived notes I need to unarchive them. 
- Will I find them in a folder of now 60+ notes? Will have to put them back into the archive. 
- I should have labelled them "archive" before archiving them. Or have them moved into a subfolder of my topic's folder so I could unarchive them without any hassle. 
- And call that folder "Archive". 
- Actually, if I did that I would not have to archive/unarchive them and had all options available to do with them whatever I want.
- What if every top level folder would be created containing a nested "Archive" folder already? 
- What if it also contained a folder called "Trash"? And one "Drafts"? 
- What if the users could set up a template for the folder set s/he wants created for every top level folder?

-------

Do we need the action-restricted current archives or could nested folders be the solution?

**Comment by alialbaali** [comment]
> I'd like to outline my deliberations and will come to something a tad off topic.
> 
>     * In order to read my archived notes I need to unarchive them.

Do you need to read & edit archived notes, or just read them? I can implement a way to read the archived notes only, and not edit. Since, I believe it would defeat the purpose of an "Archive".

Regarding your other suggestions, it's a nice concept. However, it needs more studying to be implemented correctly.

**Comment by uli-on** [comment]
> Do you need to read & edit archived notes, or just read them?

Well currently I can't do anything with an archived note, so reading would be a step forward. Optimally, we could do with archived notes what we can do with normal ones. But the next step forward would maybe allow to add (and remove) labels to/from notes inside the archive, at least give them a label that allows identification when they're unarchived in their home folder, in order to put them back into the archive after editing them in some way that's possible only outside the archive at that point in time. 

Desirable would also be to copy/move them to folders other than their home folder. Generally, everything we can do in real life will be great, and I don't feel adding an annotation to an archived note in real life would be against the idea of an archive.

**Comment by uli-on** [comment]
> Regarding your other suggestions, it's a nice concept. However, it needs more studying to be implemented correctly.

The most important thing it needed right now, in my innocent thinking, is being able to move notes freely around in nested folders: From a surrounding folder "PARENT" into the nested folder "CHILD"/Archive which isn't possible right now! This can only be achieved with a workaround, with the help of an interim folder "PARENT'S SIBLING" and from there into the CHILD/Archive folder, and I think this redundant extra step out of PARENT needed fixing.
